www/runtest.php

```diff
@@ -1187,8 +1189,7 @@ function ValidateScript(&$script, &$error)
 {
     global $test;
     $url = null;
-    if (stripos($script, 'webdriver.Builder(') === false &&
-            stripos($script, '#!/webdriver') === false) {
+    if (!$req_webdriver) {
```

Comment:
The first condition should be put back in place. That was there before I added the #! hack. It's needed because webpagetest supports webdriver when used with the node.js agent and the first condition is a hack for that.